projects/documentation/.eleventy.cjs

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I've seen this done at the core of the project rather than the dependencies and config nested inside a documentation folder and it was rather nice and integrated.  If you put this at the root of the project, you can point it to compile and render assets in the documentation folder.  That said, both work great so definitely just a preference